certified-connectors/Dime.Scheduler/README.md

```diff
@@ -0,0 +1,58 @@
+![Dime.Scheduler](https://cdn.dimescheduler.com/dime-scheduler/Dime.Scheduler-Black.png)
+
+# Dime.Scheduler
```

Comment:
@hbulens please change to Publisher: Dime.Scheduler.
Please follow and include all required fields from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)

Reply:
The docs state the following:
> "publisher" is the name of your company or organization. Provide the full company name (for example, "Contoso Corporation"). 

Hence, Dime Software should be the only correct value as it's the company that owns Dime.Scheduler, which is one of the products that we own and manage.

Reply:
@hbulens yes you are correct. but please add the Publisher section in the readme.
Your readme file has the following.
```
# Dime.Scheduler

Dime.Scheduler, a Dime Software product, is a resource and project planning solution for the Microsoft Dynamics product suite.
```
Please add the publisher section to be 
```
# Dime.Scheduler

Dime.Scheduler, a Dime Software product, is a resource and project planning solution for the Microsoft Dynamics product suite.

#Publisher: Dime.Scheduler
```
and please add all required sections from this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). The readme file is missing the below sections.
```
# Obtaining Credentials
# Known Issues and Limitations
# Deployment Instructions
```

I hope this helps.

Please let me know if you have any questions.

Thank you so much for your hard work with us.

Reply:
Thanks for clarifying, I did not understand your remarks referred to the readme file. Should be better now with commit [3441f57](https://github.com/microsoft/PowerPlatformConnectors/pull/2091/commits/3441f5768f2b5d1996f988a81934797fc1d7b7f5).